backend/event-service/src/main/kotlin/com/ticketqueue/event/service/HallService.kt

```
@@ -0,0 +1,202 @@
+     * @param venueId 공연장 ID
+     * @param hallId 홀 ID
+     * @return 홀 상세 정보 (좌석 템플릿 포함)
+     * @throws EventException 홀이 존재하지 않거나 (HALL_NOT_FOUND),
+     *         좌석 템플릿 데이터가 손상된 경우 (INVALID_SEAT_TEMPLATE)
+     */
+    fun getHall(venueId: UUID, hallId: UUID): HallDto.DetailResponse {
+        val hall = findHallByVenueIdAndId(venueId, hallId)
+        // JSONB 문자열 -> SeatTemplateDto 역직렬화 (에러 핸들링)
+        val seatTemplate = try {
+            objectMapper.readValue(hall.seatTemplate, SeatTemplateDto::class.java)
+        } catch (e: JsonProcessingException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)